src/Hosting/Abstractions/src/IHostApplicationLifetime.cs

```diff
@@ -16,14 +16,14 @@ public interface IHostApplicationLifetime
         CancellationToken ApplicationStarted { get; }
 
         /// <summary>
-        /// Triggered when the application host is performing a graceful shutdown.
-        /// Shutdown will block until this event completes.
+        /// Triggered when the application host is beginning a graceful shutdown.
```

Comment:
```suggestion
        /// Triggered when the application host is starting a graceful shutdown.
```

nit: beginning -> starting (reads better IMO)

Reply:
Lol!  I deliberately chose 'beginning' rather than 'starting' because I felt there was a jar between the 'stopping' in the property name and having 'starting' in the description.   "It's starting the stopping but you can stop the stopping before it starts if you stop in the callback."
But obviously I will defer to you!